controller_interface/src/controller_interface.cpp

```diff
@@ -174,4 +178,4 @@ std::shared_ptr<rclcpp::Node> ControllerInterface::get_node()
   return node_;
 }
 
-}  // namespace controller_interface
+}  // namespace controller_interface
```

Comment:
please dont' remove the newline, there's no point to it

Reply:
there is some bad merge happened to the repo. I will correct everything.